src/net/sourceforge/kolmafia/persistence/WitchessSolutionDatabase.java

```diff
@@ -0,0 +1,132 @@
+package net.sourceforge.kolmafia.persistence;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.*;
```

Comment:
we tend to put all the imports and not * them in src/

Reply:
Apologies, apparently my IDE automatically changed some imports to .* when I had enough things imported from them, I'll manually add only the required imports.